content/blog/2020-07-10-july-20-dvc-heartbeat.md

```diff
@@ -0,0 +1,150 @@
+---
+title: July '20 Heartbeat
+date: 2020-07-10
+description: |
+  Catch up on new DVC releases, talks, and projects in our community. 
+  This month, we recap the DVC 1.0 release, making the list of top 20 fastest growing open-source startups, and interviews galore.
+descriptionLong: |
+  Catch up on new DVC releases, talks, and projects in our community. 
+  This month, we recap the DVC 1.0 release, making the list of top 20 fastest growing open-source startups, and interviews galore.
+picture: 2020-07-10/july_20_heartbeat_header.png
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/june-20-heartbeat/404
+tags:
+  - Heartbeat
+  - GitHub Actions
```

Comment:
let's do CI/CD? we probably already have one and it's not about GIthub only as far as I can tell

Reply:
done